modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -380,7 +380,7 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
       if (_.isUndefined(prebuild.blockHeight)) {
         prebuild.blockHeight = (yield self.getLatestBlockHeight()) as number;
       }
-      transaction.locktime = prebuild.blockHeight + 1;
+      transaction.locktime = prebuild.blockHeight;
```

Comment:
let's add a comment describing why this is done

we could either copy or link to https://github.com/bitcoin/bitcoin/blob/fb0ac482eee761ec17ed2c11df11e054347a026d/src/wallet/wallet.cpp#L2133

Reply:
I've added a comment.